plugin/src/main/kotlin/com/vanniktech/maven/publish/Platform.kt

```diff
@@ -430,6 +425,25 @@ private fun MavenPublication.withJavadocJar(factory: () -> TaskProvider<*>?) {
   }
 }
 
+private fun setupTestFixtures(project: Project, sourcesJar: Boolean) {
+  project.plugins.withId("java-test-fixtures") {
+    if (sourcesJar) {
+      // TODO: remove after https://github.com/gradle/gradle/issues/20539 is resolved
+      project.serviceOf<JvmModelingServices>().createJvmVariant("testFixtures") {
+        it.withSourcesJar().published()
+      }
+    }
+
+    // test fixtures can't be mapped to the POM because there is no equivalent concept in Maven
+    project.mavenPublications {
+      it.suppressPomMetadataWarningsFor("testFixturesApiElements")
+      it.suppressPomMetadataWarningsFor("testFixturesRuntimeElements")
+      it.suppressPomMetadataWarningsFor("testFixturesSourcesElements")
```

Comment:
This last one isn't present in eithernet because it gets added by the `it.withSourcesJar().published()` above which didn't run on eithernet since it's a Kotlin project